bin/options.js

```diff
@@ -62,17 +62,6 @@ const options = {
     describe: 'Open default browser with the specified page',
     requiresArg: true,
   },
-  color: {
```

Comment:
I removed the `color` CLI flag since it ends up setting `colors: true` in the stats option, which was also removed.

New behavior:
- default for `colors` is `false` (should it be `true`?)
- if a user wants to turn on color, they need to put in their webpack config: `stats: { colors: true }`.

Do you think we should do this approach, or should we keep the color CLI flag and then update the compiler stats option using this flag?

Reply:
I think we don't need this option, default value is `stats.colors` (https://github.com/webpack/webpack/blob/master/lib/stats/DefaultStatsPresetPlugin.js#L168), `webpack-cli` should have `--stats-color` (or `--color`) flag to enable or disable it, it is out of scope webpack-dev-server